services/_system.md

```diff
@@ -48,6 +48,16 @@ Event from sensor or a broadcast service.
 
 Request to calibrate a sensor. The report indicates the calibration is done.
 
+    report command_not_implemented @ 0x03 {
```

Comment:
"command" is a bit misleading here since it also applies to "register set/get"

Reply:
register get/set is a command; the other thing is `action`; we should update our spec language, but the protocol docs are updated I think